shared/templates/template_OVAL_audit_rules_unsuccessful_file_modification

```diff
@@ -52,7 +52,7 @@
       <value>^[\s]*-a[\s]+always,exit[\s]+(?:-F[\s]+arch=b64[\s]+)(?:.*(-S[\s]+{{{ NAME }}}[\s]+|([\s]+|[,]){{{ NAME }}}([\s]+|[,]))).*</value>
   </constant_variable>
   <constant_variable id="var_arufm_{{{ NAME }}}_tail" version="1" datatype="string" comment="audit rule auid and key">
-    <value>[\s]+(?:-F\s+auid>={{{ auid }}}[\s]+)(?:-F\s+auid!=(unset|4294967295)[\s]+)(?:-k[\s]+|-F[\s]+key=)[\S]+[\s]*$</value>
+    <value>[\s]+(?:-F\s+auid>={{{ auid }}}[\s]+)(?:-F\s+auid!=(unset|4294967295)[\s]+)(?:-k[\s]+|-F[\s]+key=)unsuccesful-access[\s]*$</value>
```

Comment:
This hardcode key won't align with all rules that use this template, see for example these other rules:
- delete: [audit_rules_unsuccessful_file_modification_unlink](https://github.com/ComplianceAsCode/content/blob/f42bc985ea3ec1704ae561634baa2d4e5920601e/linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_unlink/rule.yml)
- permission change: [audit_rules_unsuccessful_file_modification_fchown](https://github.com/ComplianceAsCode/content/blob/f42bc985ea3ec1704ae561634baa2d4e5920601e/linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_fchown/rule.yml)